### PR TITLE
add configmap for builder, deployer image data

### PR DESCRIPTION
**Pull request by juanvallejo**
cc @deads2k

### CODE REVIEW

File: manifests/0000_21_cluster-openshift-controller-manager-operator_03_builder-deployer-config.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-core-operators
+  name: openshift-cluster-openshift-controller-manager-operator-builder-deployer-config
```
Comment (deads2k) [inline manifests/0000_21_cluster-openshift-controller-manager-operator_03_builder-deployer-config.yaml]: ah, you learn names from the best.  @liggitt @sttts 

In this case though, how about `openshift-controller-manager-images`?

**Comment by deads2k** [comment]
can you wire up the require image references file entry for these too?

**Comment by juanvallejo** [comment]
@deads2k thanks, done. As for the builder image, I was only able to find [https://quay.io/repository/openshift/origin-docker-builder](origin-docker-builder) on quay.io.

I did find origin-deployer on quay.io

File: manifests/0000_21_cluster-openshift-controller-manager-operator_03_builder-deployer-config.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-core-operators
+  name: openshift-controller-manager-images
+data:
+  builderImage: quay.io/openshift/origin-docker-builder:v4.0
```
Comment (deads2k) [inline manifests/0000_21_cluster-openshift-controller-manager-operator_03_builder-deployer-config.yaml]: @bparees is this image the right image for the builder?

**Comment by deads2k** [comment]
@juanvallejo get confirmation from @bparees  and this  lgtm (you can tag)

/approve

File: manifests/0000_21_cluster-openshift-controller-manager-operator_03_builder-deployer-config.yaml (hunk 2 above)
Comment (bparees) [reply manifests/0000_21_cluster-openshift-controller-manager-operator_03_builder-deployer-config.yaml]: yes

**Pull request merged**